stores/current.ts

```diff
@@ -159,29 +159,51 @@ function round2(value: number): number {
 }
 
 function round4(value: number): number {
+  if (typeof value !== 'number') {
```

Comment:
Can you remove this check? We use TypeScript to check argument type.

This warning take big part of JS bundle and never will be called.